hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/utils/TestData.java

```diff
@@ -503,6 +503,20 @@ public static String rowDataToString(List<RowData> rows) {
   public static void writeData(
       List<RowData> dataBuffer,
       Configuration conf) throws Exception {
+    writeData(dataBuffer, conf, 1);
+  }
+
+  /**
+   * Write a list of row data with Hoodie format base on the given configuration.
+   *
+   * @param dataBuffer The data buffer to write
+   * @param conf       The flink configuration
+   * @param ckpId      The checkpoint id
+   * @throws Exception if error occurs
+   */
+  public static void writeData(
+          List<RowData> dataBuffer,
```

Comment:
The checkpoint id does not affect the data write, there is no need to specify it explicitly.

Reply:
Revert this change.